src/js/components/Form/stories/CustomTheme.js

```diff
@@ -16,21 +29,60 @@ const customFormFieldTheme = {
   formField: {
     label: {
       color: 'dark-3',
-      size: 'xsmall',
-      margin: { vertical: '0', bottom: 'small', horizontal: '0' },
+      size: 'small',
+      margin: { vertical: 'none', bottom: 'small', horizontal: 'none' },
       weight: 600,
     },
     border: false,
-    margin: 0,
+    margin: { bottom: 'small' },
   },
 };
 
 const CustomFormField = () => (
-  <Grommet theme={deepMerge(grommet, customFormFieldTheme)}>
-    <Box align="center" pad="large">
-      <FormField label="Label" htmlFor="text-area">
-        <TextArea id="text-area" placeholder="placeholder" />
-      </FormField>
+  <Grommet full theme={deepMerge(grommet, customFormFieldTheme)}>
+    <Box fill align="center" justify="center">
+      <Box width="medium">
+        <Form
+          onReset={event => console.log(event)}
+          onSubmit={({ value }) => console.log('Submit', value)}
+        >
+          <FormField label="Name" name="name" required>
+            <TextInput name="name" />
+          </FormField>
+          {/* <FormField label="Email" name="email" required>
+            <MaskedInput
+              name="email"
+              mask={[
+                { regexp: /^[\w\-_.]+$/, placeholder: 'example' },
+                { fixed: '@' },
+                { regexp: /^[\w]+$/, placeholder: 'my' },
+                { fixed: '.' },
+                { regexp: /^[\w]+$/, placeholder: 'com' },
+              ]}
+            />
+          </FormField> */}
```

Comment:
Can we remove this potion of the code?

Reply:
I de-commented it now that MaskedInput is a hook component too.